src/components/forms/SmallGrantsForm.tsx

```diff
@@ -140,6 +142,13 @@ export const SmallGrantsForm: FC = () => {
                   </PageText>
                 </Box>
               )}
+              {errors?.firstName?.type === 'validate' && (
+                <Box mt={1}>
+                  <PageText as='small' fontSize='helpText' color='red.500'>
+                    First name contain a URL.
```

Comment:
For consistency within the error messages and more clarity, we should rephrase it as 'First name cannot contain a URL'